### PR TITLE
FEAT: `degrees` and `rad2deg` ufuncs implementation

### PR DESCRIPTION
**Pull request by SwayamInSync**
## Copilot Summary
This pull request adds support for converting radians to degrees in quad-precision and long double types, implements the corresponding ufuncs (`degrees` and `rad2deg`), and introduces comprehensive tests for these operations. It also updates the release tracker to reflect the new functionality.

**New quad-precision and long double degree conversion support:**

* Added `quad_degrees` and `ld_degrees` functions to convert radians to degrees for `Sleef_quad` and `long double` types, respectively (`quaddtype/numpy_quaddtype/src/ops.hpp`). [[1]](diffhunk://#diff-ad4140fa03d374f246bdc4d441d868fa759788f2d27a7a29ca259b16780f0850R246-R254) [[2]](diffhunk://#diff-ad4140fa03d374f246bdc4d441d868fa759788f2d27a7a29ca259b16780f0850R458-R467)

**Ufunc integration:**

* Registered the new `degrees` and `rad2deg` ufuncs for quad-precision and long double types in the NumPy extension (`quaddtype/numpy_quaddtype/src/umath/unary_ops.cpp`).

**Testing improvements:**

* Added parameterized tests to verify correctness of the `degrees` and `rad2deg` ufuncs, including edge cases and special values (`quaddtype/tests/test_quaddtype.py`).

**Documentation and tracking:**

* Updated the release tracker to indicate that `degrees` and `rad2deg` are now implemented and tested (`quaddtype/release_tracker.md`).

### CODE REVIEW

File: quaddtype/numpy_quaddtype/src/ops.hpp [hunk 1]
```diff
@@ -243,6 +243,15 @@ quad_atanh(const Sleef_quad *op)
     return Sleef_atanhq1_u10(*op);
 }
 
+static inline Sleef_quad
+quad_degrees(const Sleef_quad *op)
+{
+    // degrees = radians * 180 / π
+    static const Sleef_quad one_eighty = Sleef_strtoq("180.0", NULL);
```
Comment (juntyr) [inline quaddtype/numpy_quaddtype/src/ops.hpp]: same as the other PR

**Comment by SwayamInSync** [comment]
Merging this in!
